encoding/protobuf/protobuf_test.go

```diff
@@ -23,12 +23,11 @@ import (
 	"strings"
 	"testing"
 
-	"github.com/google/go-cmp/cmp"
-
 	"cuelang.org/go/cue/ast"
 	"cuelang.org/go/cue/errors"
 	"cuelang.org/go/cue/format"
 	"cuelang.org/go/internal/cuetest"
+	"github.com/google/go-cmp/cmp"
```

Comment:
please don't move this import

Reply:
Sorry, that was probably the auto formatter, will fix

Reply:
Fixed